datalad_container/containers_add.py

```diff
@@ -138,7 +138,8 @@ class ContainersAdd(Interface):
             the rest of the URL will be interpreted as the argument to
             'docker pull', the image will be saved to a location
             specified by `name`, and the call format will be auto-configured
-            to run docker, unless overwritten.""",
+            to run docker, unless overwritten. The auto-configured call to docker 
+            run mounts the cwd to '/tmp' and sets the working directory to '/tmp'.""",
```

Comment:
Thanks! really that is what it does? yikes -- something we might want to improve to avoid committing all the junk -- but it indeed might be good for "complete provenance collection" ;)  meanwhile since CWD is an acronym

```suggestion
            run mounts the CWD to '/tmp' and sets the working directory to '/tmp'.""",
```